pkg/codegen/testing/test/testdata/azure-native-nested-types/nodejs/documentdb/index.ts

```diff
@@ -5,10 +5,12 @@ import * as pulumi from "@pulumi/pulumi";
 import * as utilities from "../utilities";
 
 // Export members:
-export * from "./sqlResourceSqlContainer";
 
-// Import resources to register:
-import { SqlResourceSqlContainer } from "./sqlResourceSqlContainer";
+export { SqlResourceSqlContainerArgs } from "./sqlResourceSqlContainer";
+export type SqlResourceSqlContainer = import("./sqlResourceSqlContainer").SqlResourceSqlContainer;
```

Comment:
2. re-export the type of the resource class; this is needed if it's used for declaring types and type-checking